Doc/howto/unicode.rst

```diff
@@ -135,10 +135,8 @@ used than UTF-8.)  UTF-8 uses the following rules:
 UTF-8 has several convenient properties:
 
 1. It can handle any Unicode code point.
-2. A Unicode string is turned into a sequence of bytes containing no embedded zero
-   bytes.  This avoids byte-ordering issues, and means UTF-8 strings can be
-   processed by C functions such as ``strcpy()`` and sent through protocols that
-   can't handle zero bytes.
+2. A UTF-8 string is turned into a sequence of bytes that contains embedded
```

Comment:
You shouldn't change the beginning of the sentence from "A Unicode string" to "A UTF-8 string"; the original string is a Unicode string, not a UTF-8 string. If it were already a "UTF-8 string", it wouldn't be being "turned into" anything. Otherwise, looks good.